lib/aux_js.cpp

```diff
@@ -207,11 +208,11 @@ int display(const int field_type, std::stringstream & commands, const int w,
       }
       if (stream_state.mesh->SpaceDimension() == 2 && field_type == 2)
       {
-         SetVisualizationScene(vs, 2);
+         SetVisualizationScene(vs, 2, stream_state.keys.c_str());
       }
       else
       {
-         SetVisualizationScene(vs, 3);
+         SetVisualizationScene(vs, 3, stream_state.keys.c_str());
```

Comment:
This looks unrelated to the changes, is it some bugfix?

Reply:
Nope - was copying stuff over from `glvis.cpp` but this shouldn't be here. Thanks :)